score_keras.py

```diff
@@ -54,17 +57,18 @@ def load_model_and_images(model_file, image_dir, seed):
     testpath = image_dir
 
     imagegen = image.ImageDataGenerator()
-    test_gen = image.DirectoryIterator(testpath, imagegen, target_size=(224, 224), seed=seed, shuffle=True)
+    test_gen = image.DirectoryIterator(testpath, imagegen, target_size=(299, 299), seed=seed, shuffle=True)
 
     model = load_model(model_file)
     # Get classes sorted by their value
     classes = [x[0] for x in sorted(test_gen.class_indices.items(), key=lambda x: x[1])]
 
     return model, test_gen, classes
 
-def evaluate_model(model, image_gen, classes, num_batches, output_metrics, output_image, top_n=None, beta=1.):
-    for batch_num in range(num_batches):
-        logger.info('Processing batch %d of %d' % (batch_num, num_batches))
+def evaluate_model(model, image_gen, classes, output_metrics, output_image, top_n=None, beta=1.):
+    # Iterate all batches for testing
+    for batch_num in range(len(image_gen)):
+        logger.info('Processing batch %d of %d' % (batch_num, len(image_gen)))
```

Comment:
Change `Processing batch` to `Processing image`

Reply:
Do you mean current logs below don't give you enough information?
```
Found 1064 images belonging to 19 classes.
Processing batch 0 of 34
Processing batch 1 of 34
```
Maybe this would be more helpful?
```
Found 1064 images belonging to 19 classes.
Batch size is 32
Processing batch 0 of 34
Processing batch 1 of 34
```

Reply:
No, perhaps I'm misunderstanding the generator. It looked to me like you were processing the images one at a time. If so, just wanted you to change the word "batch" to "image" to reflect that.